src/SvgIcon/SvgIcon.js

```diff
@@ -0,0 +1,72 @@
+// @flow weak
+
+import React, { PropTypes } from 'react';
+import { createStyleSheet } from 'jss-theme-reactor';
+import classNames from 'classnames';
+import { easing } from '../styles/transitions';
+
+export const styleSheet = createStyleSheet('SvgIcon', (theme) => {
+  return {
+    SvgIcon: {
```

Comment:
I don't think that we have a convention on that point. I'm used to use `root` but I don't have much preference.
@nathanmarks Could argue in one direction.